rllib/examples/quadx_waypoints.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import os
+
+from ray.rllib.examples.envs.classes.pyflyt_quadx_waypoints_env import (
+    create_quadx_waypoints_env,
+)
+from ray.rllib.utils.test_utils import check_learning_achieved
+from ray.tune.registry import get_trainable_cls
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--run", type=str, default="PPO", help="The RLlib-registered algorithm to use."
+)
+parser.add_argument("--num-cpus", type=int, default=4)
+parser.add_argument(
+    "--framework",
```

Comment:
We are moving to a new API stack that supports our `RLModule API`, `Learner API`, and `EnvRunner API`, mainly for better customization and performance. Therein, we support mainly PyTorch (for PPO and IMPALA also `tf2` but no `tf` anymore). 

Could we adjust this arg towards this?